src/include/rados/librados.hpp

```diff
@@ -1318,7 +1318,7 @@ namespace librados
       const std::string& name,     ///< daemon name (e.g., 'gwfoo')
       const std::map<std::string,std::string>& metadata); ///< static metadata about daemon
     int service_daemon_update_status(
-      const std::map<std::string,std::string>& status);
+      std::map<std::string,std::string>&& status);
```

Comment:
the ABI change is discussed at https://github.com/ceph/ceph/pull/18468#discussion_r146177293